frontend/src/ui/Blocks/Series.tsx

```diff
@@ -394,6 +792,14 @@ const GridTile: React.FC<GridTypeProps> = ({ event, basePath, active }) => {
                 boxShadow: "0 0 0 2px var(--accent-color)",
             },
         },
+        // ListView styles:
```

Comment:
List view on mobile is broken still: the description and all that stuff needs to go under the thumbnail on small screens.